hanami-router.gemspec

```diff
@@ -20,10 +20,12 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.2.0'
 
   spec.add_dependency 'http_router',  '~> 0.11'
+  spec.add_dependency 'oj',           '~> 2.15.0' unless RUBY_PLATFORM == 'java'
```

Comment:
I'm thinking that with the option to specify which JSON parser to use, we can drop the hard dependency on Oj and let application developers include it on their own.